### PR TITLE
Cancelled/Rejected API

### LINKED ISSUE #86: Cancelled page API
get all the cancelled request by the warden

### PR DESCRIPTION
**Pull request by ayushSinha16**

### CODE REVIEW

File: server-side/node-mssql-crud/src/controllers/warden.controller.js [hunk 1]
```diff
@@ -22,4 +22,21 @@ export const gatepassApproveOrReject = async (req, res) => {
...
+export const gatepassCancelAndReject = async(req,res) => {
+    const approval_to = req.user.data.user_id
+    try{
+        const pool= await getConnection();
+        const result = await pool
+        .request()
+        .input("approval_to",sql.VarChar,approval_to)
+        .query(queries.getRejectedAndCancelledGatepass)
+
+    return res.json(result.recordsets[0])
```
Comment (palashmunshi) [inline server-side/node-mssql-crud/src/controllers/warden.controller.js]: is this working (recordsets[0]) or should it be result.recodset ??

**Review by palashmunshi — CHANGES_REQUESTED** [review]

**Comment by palashmunshi** [comment]
Also this branch has merge conflicts. Please pull the latest localfixed, then merge localfixed into this and resolve the conflicts and then push

File: server-side/node-mssql-crud/src/controllers/warden.controller.js (hunk 1 above)
Comment (ayushSinha16) [reply server-side/node-mssql-crud/src/controllers/warden.controller.js]: It works, but for sakes of simplicity and overall code similarity, changed it to result.recordset

Comment (palashmunshi) [reply server-side/node-mssql-crud/src/controllers/warden.controller.js]: result.recordset also works right??? please test once

**Comment by ayushSinha16** [comment]
done

**Review by palashmunshi — APPROVED** [review]
nice

**Pull request merged**